setup.py

```
@@ -1,4 +1,12 @@
 from distutils.core import setup
+from subprocess import call
+
+
+def generate_requirements():
+    with open('requirements.txt', 'w') as f:
+        call(['pipenv', 'lock', '--requirements'], stdout=f)
```

call() will fail silently on errors, we would like to fail loudly in this case.